### ISSUE TITLE
Firmware/chipset cannot be selected after installation

### ISSUE THREAD

**Issue by mirh** [virt-manager/virt-manager#245]
*I see* why such a big change is schemed to be one-time-only
... though at the same time, if not any the way the "lack of selection" is currently presented, can easily suggest to the casual user VM/QEMU/virt doesn't support anything else. 

Either make it clearer (it's not a big deal if I have to create a new vm), or allow further selection.

**Comment by Ghosthree3** [comment]
This got me too, it took me months to figure out why the drop down select was gone. You can actually edit the xml directly to change both and it doesn't break the machine or anything (I've personally changed it from UEFI to BIOS back to UEFI and didn't end up with a bricked VM), I see no reason why it shouldn't be changeable by the GUI. If it's considered unsafe, add a hazard icon with a tooltip?

**Comment by crobinso** [comment]
I agree it would be a nice addition to report somewhere in the UI that the field is only changeable at install time.

For linux at least, switching from BIOS to UEFI is unlikely to work, unless the machine was originally provisioned for UEFI. I've switched old VMs to UEFI manually but it requires a bunch of non-trivial configuration changes inside the VM. Maybe if the VM was provisioned for UEFI you can switch back and forth with the BIOS. Regardless, per (DESIGN.md)(https://github.com/virt-manager/virt-manager/blob/master/DESIGN.md) I consider this an advanced usecase, one that's not too hard to work around manually.

To switch from BIOS to UEFI you can use `sudo virt-xml $VMNAME --edit --confirm --boot uefi`
Switching back to bios is a bit less clear with `sudo virt-xml uefi --edit --confirm --xml xpath.delete=./os/loader`

**Comment by mirh** [comment]
Switching boot scheme requires to the very least a switch in the bootloader mechanism on the disk
Then, putting aside microsoft kinda made this a breeze for windows with MBR2GPT, my use case was actually booting off differently partitioned disk in the same VM. 
Though, I guess like in retrospect a "duplicate this VM" button could do it as well.
